release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -480,6 +486,16 @@ revision "2023-06-30" {
        "RFC 791: Internet Protocol";
     }
 
+   leaf gratuitous-arp-accepted {
+     type boolean;
+     default false;
```

Comment:
I would suggest removing the default, unless you have a strong argument for the `false` value.

Reply:
Done